### PR TITLE
Add Manage Citations button near Edit Scenario button

### PR DESCRIPTION
**Pull request by amedenblik**
Fixes issue #89. 

I set up a "Manage Citations" button and modal that shows all citations based on the filtering in the sidebar. Service endpoint does not exist yet, so the citations cannot actually be "managed" yet.

### CODE REVIEW

**Review by harper-wavra — APPROVED** [review]
Looks good to me!

File: src/app/shared/components/managecitations/managecitations.component.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            }
+        });
+        this.modalSubscript = this._nssService.showManageCitationsModal.subscribe((show: boolean) => {
+            if (show) { this.showModal(); }
+        });
+        this._nssService.selectedRegion.subscribe(region => {
+            this.selectedRegion = region;
+            if (region && region.id) {this.getRegRegions(); }
+        });
+        this._nssService.getVersion.subscribe((v: string) => {
```
Comment (kjacobsen16) [inline src/app/shared/components/managecitations/managecitations.component.ts]: I think this can be taken out as well.

Comment (amedenblik) [reply src/app/shared/components/managecitations/managecitations.component.ts]: Which lines?

**Review by kjacobsen16 — CHANGES_REQUESTED** [review]
This looks great! I left one comment - normally I wouldn't mention it, but this can be a learning experience with pull requests that you have to make changes on!

File: src/app/shared/components/managecitations/managecitations.component.ts (hunk 1 above)
Comment (kjacobsen16) [reply src/app/shared/components/managecitations/managecitations.component.ts]: Oh yeah, that looks confusing. Starting at 59, that function where it subscribes to the appVersion. I don't think the appVersion is used. You can take that out of the variables as well.

**Review by kjacobsen16 — APPROVED** [review]
Looks good! 👍

**Pull request merged**